ocaml/xapi/xapi_globs.ml

```diff
@@ -931,8 +931,7 @@ let winbind_machine_pwd_timeout = ref (7 * 24 * 3600)
 
 let winbind_update_closest_kdc_interval = ref (3600. *. 24.) (* every day *)
 
-let winbind_kerberos_encryption_type =
-  ref Kerberos_encryption_types.Winbind.Strong
+let winbind_kerberos_encryption_type = ref Kerberos_encryption_types.Winbind.All
```

Comment:
> * Some old DC only support legacy encryption type
> * One way trust needs extra configration in DC to support strong encrytion type

Can we add this to the documentation instead of downgrading the crypto being used?

Reply:
We will document about this, 
but I am afraid we cannot avoid downgrading the crypto being used here, otherwise it will break our own usage (and thus xenrt) as there is one citrite domain (maybe ROOTCTX, not very sure) only support legacy encrytion type.
also, one way trust is used in citrite (eng.citrite.net ----one way trust--> citrite.net)

Reply:
I am afraid we have to enable `All` ciphers here, as
- One way trust needs this (otherwise, extra AD configuration is required)
- Customer may upgrade to next, and have their environment broken.  (PBIS do support legacy).
BTW, it is agreed during design review meeting with security team we enable all, and document how to improve security by set to `Strong`.  
per my understand,  this is only about cyphers,  kerberos protocol improve the security also.

Reply:
@liulinC The requirement says 5 cihpers but not All - is the Winbind.All exactly equaling below?
Also I think the requirement indicate that we should use the AES first (over the other 3 legacy ciphers), how is this matching the requirement?

- aes256-cts-hmac-sha1-96
- aes128-cts-hmac-sha1-96
- RC4-HMAC
- DES-CBC-CRC
- DES-CBC-MD5

Reply:
I think we have talked about this in the desgin. 
- all means the 5 ciphers, 
- strong means only aes256-cts-hmac-sha1-96 and aes128-cts-hmac-sha1-96
- legacy means only RC4-HMAC etc,
Winbind negotiate ciphers with DC, and will pick aes as possible, but for cases like one-way trust and only old ciphers, it will fallback to legacy.